src/stateMachine/HotFire/HotFireStateMachine.cpp

```diff
@@ -807,27 +884,30 @@ void HotFireStateMachine::detectExternEvent(const std::shared_ptr<StateData> &da
         IgnitionEXT();
         break;
     case 3:
-        FinalVentingEXT();
+        IgnitionBurnEXT();
         break;
     case 4:
-        DoneEXT();
+        FinalVentingEXT();
         break;
     case 5:
-        AbortEXT();
+        DoneEXT();
         break;
     case 6:
-        ServoControlEXT();
+        AbortEXT();
         break;
     case 7:
-        ReadyEXT();
+        ServoControlEXT();
         break;
     case 8:
-        heaterOn = true;
+        ReadyEXT();
         break;
     case 9:
-        heaterOn = false;
+        heaterOn = true;
         break;
     case 10:
+        heaterOn = false;
+        break;
+    case 11:
```

Comment:
No, keep the IDs the same, just use a different order. Ground station can handle it